xbmc/windows/GUIWindowSystemInfo.cpp

```diff
@@ -109,7 +109,7 @@ void CGUIWindowSystemInfo::FrameMove()
     if (m_diskUsage.empty())
       m_diskUsage = CServiceBroker::GetMediaManager().GetDiskUsage();
 
-    for (size_t d = 0; d < m_diskUsage.size(); d++)
+    for (size_t d = 0; d < m_diskUsage.size() && d <= 13; d++)
```

Comment:
maybe set this magic number to some constexpr int with a meaningful description? Had to look into your PR description to understand the value :)

Reply:
Updated.